src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -9,6 +9,12 @@
 
   private WebDriver driver;
 
+  /**
+   * Adding default testing group is invoked explicitly in URL is temporary change until
+   * the end of experiment (19.04.2016). Experiment is described in DAT-4052.
+   */
+  private static final String defaultTestingGroup = "AbTest.FAN_KNOWLEDGE_MERCURY_GLOBAL_NAV=DEFAULT";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)